server/channels/api4/permission.go

```diff
@@ -12,7 +12,9 @@ import (
 )
 
 func (api *API) InitPermissions() {
+	// to be deprecated - kept for backward compatibility
 	api.BaseRoutes.Permissions.Handle("/ancillary", api.APISessionRequired(appendAncillaryPermissions)).Methods("GET")
```

Comment:
Is there a follow-up ticket for removing the `GET` endpoint that we can link in the code?

Reply:
PR is created...will merge once v10 opens for merging. 
https://github.com/mattermost/mattermost/pull/27605